tools/miqversions.rb

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env ruby --disable-gems
+#
+# When run as a program, prints the following table
+#
+#     $ tools/miqversions.rb
+#     +----------------+------------------------------+------------+-------+-------+------------+
+#     | MANAGEIQ       | CLOUDFORMS MANAGEMENT ENGINE | CLOUDFORMS | RUBY  | RAILS | POSTGRESQL |
+#     +----------------+------------------------------+------------+-------+-------+------------+
+#     |                | 5.1.z                        | 2.0        |       |       |            |
+#     |                | 5.2.z                        | 3.0        |       |       |            |
+#     | Anand          | 5.3.z                        | 3.1        |       |       |            |
+#     | Botvinnik      | 5.4.z                        | 3.1        |       |       |            |
+#     | Capablanca     | 5.5.z                        | 4.0        | 2.2.z | 4.2.z | 9.4.z      |
+#     | Darga          | 5.6.z                        | 4.1        | 2.2.z | 5.0.z | 9.4.z      |
+#     | Euwe           | 5.7.z                        | 4.1        | 2.3.z | 5.0.z | 9.5.z      |
+#     | Fine           | 5.8.z                        | 4.5        | 2.3.z | 5.0.z | 9.5.z      |
+#     | Gaprindashvili | 5.9.z                        | 4.6        | 2.3.z | 5.0.z | 9.5.z      |
+#     | Hammer         | 5.10.z                       | 4.7        | 2.4.z | 5.0.z | 9.5.z      |
+#     | Ivanchuk       | 5.11.z                       | 5.0        | 2.5.z | 5.1.z | 10.y       |
+#     | Jansa          | 5.12.z                       | 5.1        | 2.5.z | 5.2.z | 10.y       |
+#     +----------------+------------------------------+------------+-------+-------+------------+
+#
+# Otherwise, it can be required in a ruby script or rake task and manipulated as needed:
+#
+#     require 'tools/miqversions'
+#
+#     MIQ::Versions.first.cfme_release
+#     #=> "5.1.z"
+#     MIQ::Versions.first.cloud_forms_release
+#     #=> "2.0"
+#     MIQ::Versions.last.miq_release
+#     #=> "Jansa"
+#     MIQ::Versions.last.ruby
+#     #=> "2.5.z"
+#     MIQ::Versions.last
+#     #=> #<struct MIQ::Version miq_release="Jansa", cfme_release="5.12.z", cloud_forms_release="5.1", ruby="2.5.z", rails="5.2.z", postrgresql="10.y">
+#
+
+module MIQ
+  Version = Struct.new(:miq_release, :cfme_release, :cloud_forms_release, :ruby, :rails, :postrgresql)
+
+  class Versions
+    extend Enumerable
+
+    # rubocop:disable Layout/ExtraSpacing, Layout/SpaceAroundOperators, Layout/IndentFirstArrayElement
+    # rubocop:disable Layout/SpaceInsideArrayPercentLiteral, Layout/SpaceInsidePercentLiteralDelimiters
+    FIELDS   = [
+         "MANAGEIQ",      "CLOUDFORMS MANAGEMENT ENGINE", "CLOUDFORMS", "RUBY", "RAILS", "POSTGRESQL"
+    ].freeze
+    VERSIONS = [
+      %w[ N/A              5.1.z                           2.0           N/A     N/A      N/A        ],
```

Comment:
Cloudforms and CFME should not appear in the ManageIQ project.

Reply:
Mentioned this elsewhere, but in case anyone stops into this PR only, I will rephrase what I said here as well:

I don't think this script provides any value without displaying all three versions in it, as personally find that to be the most confusing quirk of this project that this script and it's predecessor are trying to solve.  I understand that there are #reasons™ why we don't want to have these other products here:

- Pain to manage versions for upstream products here
- Blurring the line of what this project is
- etc. etc. etc.

And I won't contest those points again here, though I admittedly have my own personal opinions on the matter.  That said, if this is a hard stop from getting this merged, then I think this PR should be closed.  Hard stop.

If you wish to re-implement this somewhere else, you have my permission to do so, though I won't be hosting it as a project in a similar fashion that was done for `chrisarcand/miqversions` as think there are many problems that I was trying to avoid by having that owned here.

Reply:
This tool is only useful if it has all the versions, so maybe we could ignore that?

Especially given that we've never really lost all of the cfme/cloudforms references from the code base anyway? (1 cloudforms and 39 cfme references in manageiq/ alone)